cwf/protos/ue_sim.proto

```diff
@@ -34,6 +34,15 @@ message UEConfig {
 
     // Sequence Number (SEQ).
     uint64 seq = 4;
+
+    // MSISDN
```

Comment:
lets put those parameters inside an object so we can differenciate them from the standard configuration. Maybe `HSSless_config` object

Reply:
I think this proto files encapsulates the ue information which should be the same for with/without hss. Hence i added the parameters in the same object which will make the file more readable.